src/Workspaces/CSharp/Portable/Extensions/MemberDeclarationSyntaxExtensions.cs

```diff
@@ -331,7 +331,10 @@ public static ArrowExpressionClauseSyntax GetExpressionBody(this MemberDeclarati
             switch (memberDeclaration?.Kind())
             {
                 case SyntaxKind.PropertyDeclaration:
-                    return ((PropertyDeclarationSyntax)memberDeclaration).ExpressionBody;
+                    var propertyDeclaration = (PropertyDeclarationSyntax)memberDeclaration;
+                    return propertyDeclaration.ExpressionBody ??
+                        propertyDeclaration.AccessorList.Accessors.FirstOrDefault(a => a.ExpressionBody != null)
```

Comment:
Im' not sure what you're referring to.  I don't see how GetExpressionBody can be rewritten to use GetEnclosingSymbol.